global_config/production_aks.sh

```diff
@@ -1,7 +1,6 @@
-CONFIG=production
+CONFIG=production_aks
```

Comment:
This may have broken the domains:
```
$ make domains production_aks domains-plan VERBOSE=1
make: 'domains' is up to date.
make: Nothing to be done for 'production_aks'.
[ "" != "true" ] && az account set -s s189-teacher-services-cloud-production || true
./bin/terrafile -p terraform/domains/environment_domains/vendor/modules -f terraform/domains/environment_domains/config/production_aks_Terrafile
Terrafile: version 0.8, commit af25b7556a02c895ee1fce45859cb09fc238872c, built at 2023-02-16T14:10:42Z 
FATA[0000] failed to read configuration in file terraform/domains/environment_domains/config/production_aks_Terrafile due to error: open terraform/domains/environment_domains/config/production_aks_Terrafile: no such file or directory 
make: *** [Makefile:229: domains-init] Error 1
```

Reply:
fixed